eclcli/mlb/v1/mlb_policy.py

```diff
@@ -191,6 +207,22 @@ def get_parser(self, prog_name):
             help=_('Source NAT setting of the policy'),
         )
 
+        parser.add_argument(
+            '--server-name-indications',
+            metavar="<server-name=server-name,"
+                    "input-type=input-type,"
+                    "priority=priority,"
+                    "certificate-id=certificate-id>",
+            action='store',
+            nargs='+',
+            help=_("The list of Server Name Indications (SNIs) for the policy."
+                   "server-name: The server name of Server Name Indication (SNI),"
+                   "input-type: the input type of the server name,"
```

Comment:
```suggestion
                   "input-type: The input type of the server name,"
```
前後の行も、項目説明の先頭は大文字にしているため、合わせるべきかと思われます。

Reply:
修正しました